builtins.go

```diff
@@ -1471,7 +1471,7 @@ func tomlEncodeString(s string) string {
 }
 
 // tomlEncodeKey encodes a key - returning same string if it does not need quoting,
-// otherwise return it quoted; returns empty key as ''
+// otherwise return it quoted; returns empty key as ”
```

Comment:
Looks like your editor changed `''` to `"`

Reply:
Yes. Reverted this.